desktop/cmp/dash/canvas/DashCanvasModel.js

```diff
@@ -358,11 +372,11 @@ export class DashCanvasModel extends HoistModel {
         const {viewState} = this;
 
         return this.layout.map(it => {
```

Comment:
why are the handles going into the state?? -- Not sure I understand this

Reply:
ReactGridLayout lets you set the handles globally or as part of the 'layout' object for an individual view.  This was to make it so that auto-height views don't show S and SE resize handles (since the user shouldn't be resizing autoHeight views vertically).

Reply:
That said, the resizeHandles aren't being persisted in the serialized state.